lib/rbvmomi/vim/DynamicTypeMgrManagedTypeInfo.rb

```diff
@@ -29,13 +29,24 @@ def toRbvmomiTypeHash
                    'version-id-ref' => param.version,
                  }
                end,
-               'result' => {
-                 'name' => result.name,
-                 'type-id-ref' => result.type.gsub("[]", ""),
-                 'is-array' => (result.type =~ /\[\]$/) ? true : false,
-                 'is-optional' => result.annotation.find{|a| a.name == "optional"} ? true : false,
-                 'version-id-ref' => result.version,
-               }
+               'result' => (
+               if result.nil? then
+                 {
+                     'name' => 'noname',
```

Comment:
I'm not sure this is the right thing here. I wonder if `'name' => nil,` and so on isn't more appropriate... or maybe even `'result' -> nil` or the like?

I'm open to discussion.